docs/index.md

```diff
@@ -1598,9 +1598,12 @@ Environment variables and default values:
 | `--vmwarevsphere-compute-ip`      | `VSPHERE_COMPUTE_IP`      | -                        |
 
 ## Base Operating Systems
+
```

Comment:
Bah, this is something else I was working on which accidentally got committed, and needs to be taken out before mereg.